src/tokens/hooks.test.tsx

```diff
@@ -240,10 +241,11 @@ describe('useTokensForSend', () => {
     )
 
     expect(getByTestId('tokenIDs').props.children).toEqual([
-      mockPoofTokenId,
```

Comment:
The order or this has changed because now it uses the order from `tokensWithNonZeroBalanceAndShowZeroBalanceSelector`